src/util/theme.ts

```diff
@@ -0,0 +1,170 @@
+import chalk from 'chalk'
+import * as Color from 'color'
+
+import {STANDARD_CHALK, StandardChalk, Theme} from '../interfaces/theme'
+
+// eslint-disable-next-line complexity
+function standardChalk(color: StandardChalk, text: string): string {
+  switch (color) {
+    case 'red': {
+      return chalk.red(text)
+    }
+
+    case 'blue': {
+      return chalk.blue(text)
+    }
+
+    case 'yellow': {
+      return chalk.yellow(text)
+    }
+
+    case 'green': {
+      return chalk.green(text)
+    }
+
+    case 'cyan': {
+      return chalk.cyan(text)
+    }
+
+    case 'magenta': {
+      return chalk.magenta(text)
+    }
+
+    case 'white': {
+      return chalk.white(text)
+    }
+
+    case 'black': {
+      return chalk.black(text)
+    }
+
+    case 'gray': {
+      return chalk.gray(text)
+    }
+
+    case 'blackBright': {
+      return chalk.blackBright(text)
+    }
+
+    case 'redBright': {
+      return chalk.redBright(text)
+    }
+
+    case 'greenBright': {
+      return chalk.greenBright(text)
+    }
+
+    case 'yellowBright': {
+      return chalk.yellowBright(text)
+    }
+
+    case 'blueBright': {
+      return chalk.blueBright(text)
+    }
+
+    case 'magentaBright': {
+      return chalk.magentaBright(text)
+    }
+
+    case 'cyanBright': {
+      return chalk.cyanBright(text)
+    }
+
+    case 'whiteBright': {
+      return chalk.whiteBright(text)
+    }
+
+    case 'bgBlack': {
+      return chalk.bgBlack(text)
+    }
+
+    case 'bgRed': {
+      return chalk.bgRed(text)
+    }
+
+    case 'bgGreen': {
+      return chalk.bgGreen(text)
+    }
+
+    case 'bgYellow': {
+      return chalk.bgYellow(text)
+    }
+
+    case 'bgBlue': {
+      return chalk.bgBlue(text)
+    }
+
+    case 'bgMagenta': {
+      return chalk.bgMagenta(text)
+    }
+
+    case 'bgCyan': {
+      return chalk.bgCyan(text)
+    }
+
+    case 'bgWhite': {
+      return chalk.bgWhite(text)
+    }
+
+    case 'bgGray': {
+      return chalk.bgGray(text)
+    }
+
+    case 'bgBlackBright': {
+      return chalk.bgBlackBright(text)
+    }
+
+    case 'bgRedBright': {
+      return chalk.bgRedBright(text)
+    }
+
+    case 'bgGreenBright': {
+      return chalk.bgGreenBright(text)
+    }
+
+    case 'bgYellowBright': {
+      return chalk.bgYellowBright(text)
+    }
+
+    case 'bgBlueBright': {
+      return chalk.bgBlueBright(text)
+    }
+
+    case 'bgMagentaBright': {
+      return chalk.bgMagentaBright(text)
+    }
+
+    case 'bgCyanBright': {
+      return chalk.bgCyanBright(text)
+    }
+
+    case 'bgWhiteBright': {
+      return chalk.bgWhiteBright(text)
+    }
+
+    default: {
+      return chalk.gray(text)
+    }
+  }
+}
+
+function isStandardChalk(color: any): color is StandardChalk {
+  return STANDARD_CHALK.includes(color)
+}
+
+export function colorize(color: Color | StandardChalk | undefined, text: string): string {
+  if (isStandardChalk(color)) return standardChalk(color, text)
```

Comment:
I think you can remove this function and just use `return chalk[color](text)`

Using a theme with standard chalk colors

<img width="1589" alt="image" src="https://github.com/oclif/core/assets/55927613/a5fbc342-7a52-4cad-8e06-695908739ef1">